src/components/MoneyRequestHeader.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import HeaderWithCloseButton from './HeaderWithCloseButton';
+import iouReportPropTypes from '../pages/iouReportPropTypes';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import * as ReportUtils from '../libs/ReportUtils';
+import * as Expensicons from './Icon/Expensicons';
+import Text from './Text';
+import participantPropTypes from './participantPropTypes';
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import CONST from '../CONST';
+import withWindowDimensions from './withWindowDimensions';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+import Icon from './Icon';
+import lodashGet from 'lodash/get';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: iouReportPropTypes.isRequired,
+
+    /** The policies which the user has access to and which the report could be tied to */
+    policies: PropTypes.shape({
+        /** Name of the policy */
+        name: PropTypes.string,
+    }).isRequired,
+
+    /** Personal details so we can get the ones for the report participants */
+    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+
+    /** Whether we're viewing a report with a single transaction in it */
+    isSingleTransactionView: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    isSingleTransactionView: false,
+};
+
+const MoneyRequestHeader = (props) => {
+    const formattedAmount = props.numberFormat(props.report.total / 100, {
+        style: 'currency',
+        currency: props.report.currency,
+    });
+    const isSettled = /* ReportUtils.isSettled(props.report.reportID); */ false;
```

Comment:
whys this code commented out?

Reply:
`isSettled` does not exist yet. It will be added as a part of the changes made [here](https://docs.google.com/document/d/17jFcMsAotXvsMhXivoF3u7fyzeyXkKpRGXR88Ud0oWY/edit#bookmark=id.mypo05n9x3ea). This component won't be used anywhere for the moment but it is needed for other components that will link to it. So I figured we'd have this here as an indicator of what this variable would be pulled from later. But happy to adjust however we see fit!

Reply:
oh gotcha, thought maybe it was just leftover from some testing. 

I guess maybe just adding the comment at the end of the line as a todo would be more clear?
```suggestion
    const isSettled = false; // TODO: use ReportUtils.isSettled(props.report.reportID) once that method is added
```